Xamarin.Essentials/Calendar/Calendar.android.cs

```diff
@@ -138,10 +139,8 @@ static async Task<DeviceEvent> PlatformGetEventByIdAsync(string eventId)
                         Title = cur.GetString(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.Title)),
                         Description = cur.GetString(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.Description)),
                         Location = cur.GetString(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.EventLocation)),
-                        AllDay = cur.GetInt(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.AllDay)) == 1,
```

Comment:
We should be reading this property - where there's native support for an AllDay property, we should be taking this in priority over the end date

Reply:
End Date is being set based on AllDay below. Is there a different way I should be tackling this?